opensubtitles/find.go

```diff
@@ -0,0 +1,128 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+
+// FindService provides access to the find related functions
+// in the OpenSubtitles API.
+//
+// OpenSubtitles API docs: https://www.opensubtitles.com/docs/api/html/index.htm#find
+type FindService service
+
+//FeatureOptions provide the parameters for FindService.Feature
+type FeatureOptions struct {
+	//Feature ID of the feature
+	ID string `url:"id,omitempty"`
+	//IMDB ID of the feature
+	ImdbID string `url:"imdb_id,omitempty"`
+	//TMDB ID of the feature
+	TmdbID string `url:"tmdb_id,omitempty"`
+}
+
+//Feature contains the details returned by FindService.Feature
+type Feature struct {
+	Data []FeatureData `json:"data"`
+}
+
+//AttributesFeature is the detail for a given movie or tv contains in FeatureData
+type AttributesFeature struct {
+	Title           string          `json:"title"`
+	OriginalTitle   string          `json:"original_title"`
+	Year            string          `json:"year"`
+	SubtitlesCounts SubtitlesCounts `json:"subtitles_counts"`
+	SubtitlesCount  int             `json:"subtitles_count"`
+	SeasonsCount    int             `json:"seasons_count"`
+	ParentTitle     string          `json:"parent_title"`
+	SeasonNumber    int             `json:"season_number"`
+	EpisodeNumber   interface{}     `json:"episode_number"`
+	ImdbID          int             `json:"imdb_id"`
+	TmdbID          int             `json:"tmdb_id"`
+	ParentImdbID    interface{}     `json:"parent_imdb_id"`
+	FeatureID       string          `json:"feature_id"`
+	TitleAka        []string        `json:"title_aka"`
+	FeatureType     string          `json:"feature_type"`
+	URL             string          `json:"url"`
+	ImgURL          string          `json:"img_url"`
+	Seasons         []interface{}   `json:"seasons"`
+}
+
+//FeatureData contains the detail for only a given ID, be it a tv or show
+type FeatureData struct {
+	ID         string            `json:"id"`
+	Type       string            `json:"type"`
+	Attributes AttributesFeature `json:"attributes"`
+}
+
+//Feature find details for a movie or tv serie specified by an ID.
+//OpenSubtitles API docs : https://www.opensubtitles.com/docs/api/html/index.htm#find-details-for-a-feature
+func (s *FindService) Feature(ctx context.Context, opt *FeatureOptions) (feature *Feature, resp *http.Response, err error) {
+	u := "/api/v1/find/feature"
+	u, err = addOptions(u, opt)
+	if err != nil {
+		return nil, nil, nil
+	}
+	req, err := s.client.NewRequest("GET", u, "")
+	if err != nil {
+		return nil, nil, err
+	}
+	resp, err = s.client.Do(ctx, req, &feature)
+	if err != nil {
+		return nil, resp, err
+	}
+	return
+}
+
+//FeatureOptions provide the parameters for FindService.Find
```

Comment:
Codacy found an issue: [comment on exported type FindOptions should be of the form "FindOptions ..." (with optional leading article)](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)